package.json

```diff
@@ -78,9 +64,28 @@
     "less-plugin-clean-css": "^1.5.1",
     "mini-css-extract-plugin": "^0.9.0",
     "prettier": "^1.19.1",
+    "react": "^16.8.0",
+    "react-dom": "^16.8.0",
     "terser-webpack-plugin": "^4.2.3",
     "webpack": "^4.32.2",
     "webpack-cli": "^3.3.2",
     "webpack-dev-server": "^3.4.1"
+  },
+  "peerDependencies": {
+    "@khanacademy/wonder-blocks-clickable": "^2.2.0",
+    "@khanacademy/wonder-blocks-color": "^1.1.19",
+    "@khanacademy/wonder-blocks-core": "^3.2.0",
+    "aphrodite": "^1.1.0",
+    "jquery": "^2.1.1",
+    "katex": "^0.11.1",
+    "mathquill": "git+https://git@github.com/Khan/mathquill.git#a9ae54e057c5c1acc8244a5627acbff29901d992",
+    "prop-types": "15.6.1",
+    "react": "^16.8.0",
+    "react-dom": "^16.8.0",
```

Comment:
I'm made these more lenient so that we won't get peerDep warnings in webapp when we upgrade react.  I should probably do the same with the other deps for the same reason.